style.css

```diff
@@ -0,0 +1,326 @@
+/*reset code and common starts here*/
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    text-decoration: none;
+    transition: .2s linear;
+}
+
+.btn{
+    border-radius: 0.5rem;
+    color: black;
+    background-color: rgb(223, 154, 68);
+    font-size: 20px;
+
+}
+
+.card{
+    box-shadow:4px 8px 2px grey;
+    width: 40%;
+
+}
+
+.card:hover{
+    box-shadow: 2px 4px 2px grey;
+
+}
+
+html{
+    scroll-behavior: smooth;
+   
+}
+
+/*reset code and common starts here*/
+/*nav starts here*/
+.menu__lists{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+     
+}
+
+.menu__list{
+    padding: 2rem;
+    font-size: 20px;
+}
+.menu__list:hover{
+    color: brown;
+    cursor: pointer;
+}
+
+.main-head{
+    background-color: #93C6E7;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    position: fixed;
+    top:0; left: 0; right: 0;
+    z-index: 100;
+    padding: 1rem;
+}
+.icon-ball{
+    font-size: 60px;
+    color: blue;
+}
+
+.icons{
+    padding: 1rem;
+    color: blue;
+}
+
+.icon{
+    padding: 0.5rem;
+
+}
+
+.icon:hover{
+    color: blue;
+    cursor: pointer;
+}
+/*nave ends here*/
+
+/*header starts here*/
+
+#header{
+    height: 100vh;
+    padding: 2rem;
+}
+.banner{
+    background-image: url(./images/coverpage.jpg);
+    min-height: 100%;
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: cover;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.banner__title{
+    color: #FFF2F2;
+    font-weight: bold;
+    font-size: 100px;
+}
+
+.banner__desc{
+    color: antiquewhite;
+    line-height: 1.5rem;
+    background-color: #93C6E7;
+    color: black;
+}
+
+.banner__btn{
+    border-radius: 0.3rem;
+    font-size: large;
+}
+/*header ends here*/
+
+/*main starts here*/
+
+main{
+    background: #E5E0FF;
+    min-height: 100vh;
+}
+.product-title{
+    text-align: center;
+}
+   
+
+.products{
+    margin: 20px;
+    max-width: 1200px;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns:repeat(3, 1fr);
+    grid-gap: 5px;
+    grid-auto-rows: minmax(100px, auto);
+}
+    
+.product__item{
+    border-radius: 5px;
+    overflow: hidden;
+    text-align: center;
+    box-sizing: border-box;
+    border: 1px solid #ddd; 
+    
+}
+
```

Comment:
May be here you should change it to: figure.product__item img{max-with:100%; height: auto}